src/main/java/zeenea/connector/common/ItemInventory.java

```diff
@@ -37,12 +37,29 @@ private ItemInventory(Builder builder) {
    * @param itemIdentifier the identifier for the item
    * @param labelPath the path of labels associated with the item
    * @return a new ItemInventory instance
+   * @deprecated since 2.3.0, use getLabelIdentifier() instead
    */
+  @Deprecated(
+      since =
+          "Deprecated since version 2.3.0, use getLabelIdentifier() instead. Scheduled for removal in version 3.0.0.",
```

Comment:
Ce n'est pas à getLabelIdentifier() qu'il faut se référer ici, mais plutôt à l'autre implémentation de of(). Sinon, ne pas spécifier.
```suggestion
          "Deprecated since version 2.3.0. Scheduled for removal in version 3.0.0.",
```

Reply:
Ah bien vu, c'est corrigé :)